staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness_test.go

```diff
@@ -0,0 +1,246 @@
+package filters
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	fctypesv1a1 "k8s.io/api/flowcontrol/v1alpha1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap"
+	"k8s.io/apiserver/pkg/authentication/user"
+	apifilters "k8s.io/apiserver/pkg/endpoints/filters"
+	epmetrics "k8s.io/apiserver/pkg/endpoints/metrics"
+	apirequest "k8s.io/apiserver/pkg/endpoints/request"
+	"k8s.io/apiserver/pkg/server/mux"
+	utilflowcontrol "k8s.io/apiserver/pkg/util/flowcontrol"
+	fq "k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing"
+	fcmetrics "k8s.io/apiserver/pkg/util/flowcontrol/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+const (
+	decisionNoQueuingExecute = iota
+	decisionQueuingExecute
+	decisionCancelWait
+	decisionReject
+	decisionSkipFilter
+)
+
+type fakeApfFilter struct {
+	mockDecision int
+	postEnqueue  func()
+	postDequeue  func()
+}
+
+func (t fakeApfFilter) MaintainObservations(stopCh <-chan struct{}) {
+	panic("implement me")
```

Comment:
"do nothing" would be a valid implementation here, right?

Reply:
yes, fixed.